supervision/dataset/formats/pascal_voc.py

```diff
@@ -106,11 +106,15 @@ def detections_to_pascal_voc(
             for polygon in polygons:
                 xyxy = polygon_to_xyxy(polygon=polygon)
                 next_object = object_to_pascal_voc(
-                    xyxy=xyxy, name=name, polygon=polygon
+                    xyxy=xyxy + 1,
+                    name=name,
+                    polygon=polygon,
```

Comment:
Let's move our changes to `object_to_pascal_voc` function body.
- Right now we need to make changes in multiple places. It would be better to do them in one place.
- We handled `xyxy` mutation but not `polygon`

Reply:
> Right now we need to make changes in multiple places. It would be better to do them in one place.

Makes sense! Fixed. 

> We handled xyxy mutation but not polygon

This is where I'm stuck and could use some domain knowledge. 

1. I am not sure if polygons are also 1-indexed in VOC. I downloaded the dataset and they directly provide a png file with segmentation, instead of annotations . 

2. I see we are saving polygons in `object_to_pascal_voc ` but don't see any code to load polygons. When we load pascal voc dataset, looks like we are just loading bounding boxes : https://github.com/roboflow/supervision/blob/36cdcf1b128e38cb9a086290fbfb0130fa64ac5b/supervision/dataset/formats/pascal_voc.py#L122